packages/gdl-frontend/pages/books/translations.js

```diff
@@ -333,9 +336,97 @@ const CustomGrid = styled('div')`
   }
 `;
 
-class MyTranslationsPage extends React.Component<{
-  intl: intlShape
-}> {
+const CustomSearchBar = styled('div')`
+  display: flex;
+  flex-direction: column;
+  @media (min-width: ${TABLET_BREAKPOINT}px) {
+    flex-direction: row;
+    justify-content: space-between;
+  }
+`;
+
+const CustomSearchbarItem = styled('div')`
+  width: 100%;
+  padding-top: 5px;
+`;
+
+function sortOnTitle(dir) {
+  return function(a, b) {
+    const result =
+      a.to['title'] < b.to['title']
+        ? -1
+        : a.to['title'] > b.to['title']
+        ? 1
+        : 0;
+    return result * dir;
+  };
+}
+
+function showSelectedTranslations(
+  translations,
+  refetch,
+  titleSearch,
+  sortDirection
+) {
+  if (sortDirection === 'asc') translations.sort(sortOnTitle(1));
+  else translations.sort(sortOnTitle(-1));
+
+  return translations.map(translation => {
+    if (
+      translation.to.title.toUpperCase().indexOf(titleSearch.toUpperCase()) !==
+      -1
+    ) {
+      return (
+        <TranslationCard
+          key={translation.to.id}
+          translation={translation}
+          handleSync={refetch}
+        />
+      );
+    }
+    return null;
```

Comment:
Istedenfor å returnere `null`, kanskje heller returnere et element som sier noe ala "There is no result on {titleSearch}`?